Documentation/DebugJavascriptDevTools.md

```diff
@@ -0,0 +1,29 @@
+# BabylonNative Script Debugging on Win32
+You can use Chromium DevTools to debug your JavaScript while running BabylonNative on Win32 (x86/x64) with the V8 javascript engine.
+
+## Set up BabylonNative to Use V8
+When running CMake, be sure to specify the following flag:
+`-D NAPI_JAVASCRIPT_ENGINE=V8`
```

Comment:
nit: probably should call out that you are specifying this flag to use the v8 engine which isn't used by default for win32. Although it seems somewhat obvious whats going on in the command, itd still be good to confirm that this enables v8 in the comment